CHANGELOG.md

```diff
@@ -6,10 +6,31 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.2.0] - 2020-07-09
+
+### Added
+
+- MESH API Integration
+- Partial outbound (GP->HA) Deduction transaction
+    - Only a "stub" transaction - not all fields translated to EDIFACT
+- Partial outbound (GP->HA) Removal transaction
+    - Only a "stub" transaction - not all fields translated to EDIFACT
```

Comment:
Partial outbound (GP->HA) Deduction transaction
Partial outbound (GP->HA) Removal transaction

Is it worth mentioning the fields we are translating to EDIFACT to avoid trial and error?

Reply:
Combined into a single line and listed the implemented fields